File: src/Easy-Dash/ClientApp/app/components/models/testsummary.ts
import { UrlTestStatus } from './urlteststatus';

export class TestSummary {
    summaryDescription: string = '';
    fromDate: string | Date = '';
    toDate: string | Date = '';
    numberOfTests: number = 0;
    successful: number = 0;
    failed: number = 0;
    averageDuration: number = 0;
    source: Array<UrlTestStatus> = [];
}

File: src/Easy-Dash/Models/TestSummary.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace EasyDash.Models
{
    public class TestSummary
    {
	    public string SummaryDescription { get; set; }
	    public DateTime FromDate { get; set; }
	    public DateTime ToDate{ get; set; }
	    public int NumberOfTests { get; set; }
	    public int Successful { get; set; }
	    public int Failed { get; set; }
	    public double AverageDuration { get; set; }

    }
}
